python/existing-vpc-new-ec2-ebs-userdata/cdk_vpc_ec2/cdk_vpc_ec2_stack.py

```diff
@@ -0,0 +1,54 @@
+from aws_cdk import core
+import aws_cdk.aws_ec2 as ec2
+
+vpc_id = "vpc-111111"  # Import an Exist VPC
+ec2_type = "m5.xlarge"
```

Comment:
Can this be a t2.micro? We try to bias towards free-tier eligible applications as much as possible for demo code.

Reply:
Good catch